votor/src/commitment.rs

```diff
@@ -0,0 +1,26 @@
+use {crossbeam_channel::Sender, log::trace, solana_sdk::clock::Slot};
+
+pub enum AlpenglowCommitmentType {
+    /// Our node has voted notarize for the slot
+    Notarize,
+    /// We have observed a finalization certificate for the slot
+    Finalized,
+}
+
+pub struct AlpenglowCommitmentAggregationData {
+    pub commitment_type: AlpenglowCommitmentType,
+    pub slot: Slot,
+}
+
+pub fn alpenglow_update_commitment_cache(
+    commitment_type: AlpenglowCommitmentType,
+    slot: Slot,
+    commitment_sender: &Sender<AlpenglowCommitmentAggregationData>,
+) {
+    if let Err(e) = commitment_sender.send(AlpenglowCommitmentAggregationData {
+        commitment_type,
+        slot,
+    }) {
+        trace!("commitment_sender failed: {:?}", e);
```

Comment:
should this be higher log level? Failing to send here seems bad

Reply:
good call, I bounded the channel and added some logging here 679f84bcc65478e7b57dca22c0bc3f351368096d
If the channel fails to send for disconnection, it could just be normal shutdown. In the next PR I'll use this value to set the exit flag and exit the voting loop as well.